src/investigation_segment_2_0.c

```diff
@@ -0,0 +1,138 @@
+#include "global.h"
+#include "main.h"
+#include "sound.h"
+#include "animation.h"
+
+void sub_8003594(struct Main * main)
+{
+    DmaCopy16(3, gUnknown_08015B70, gMain.roomData, sizeof(gUnknown_08015B70));
+    DmaCopy16(3, gUnknown_08015A1C, gTalkData, sizeof(gUnknown_08015A1C));
+    main->currentRoomId = 6;
+}
+
+void sub_80035D8(struct Main * main)
+{
+    DmaFill16(3, 0xFFFF, gExaminationData, sizeof(gExaminationData));
+    switch(main->currentRoomId-2) {
```

Comment:
I assume you've already tried it without the -2 while just having case 0 1 and 2 breaking or falling through to a default case right?

Reply:
I just tried this and having this without the -2 and just not including case 0 and 1 or even a default case seems to match

Reply:
resolved as of 996d70e